api/bazel/api_build_system.bzl

```diff
@@ -90,6 +90,7 @@ def api_proto_library(
         visibility = ["//visibility:private"],
         srcs = [],
         deps = [],
+        tags = [],
```

Comment:
why do you need tags even for api_proto_library? If you're not building the library/binary depends on them they're not built anyway.

Reply:
You raise a good point, we introduced tags[] wherever we observed that the underlying bazel API's supported them, but this case may not be necessary for our purpose of excluding from Windows. It might be best to support them where they are valid, though?

Reply:
We do omit some parameter as "opinionated" wrapper of underlying bazel API and add tags/options for Envoy specific. I'm not feeling strongly on this so either way is fine.